Data_Structures_Answers.md

```diff
@@ -1,18 +1,25 @@
 Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
+O(n) - driven by the for-loop search for the oldest item
```

Comment:
This can be done with `O(1)` since we are inserting into the list by index. Even if we insert at the front of the storage list, we're overwriting the value that's currently there; there is no need to shift the elements that come afterwards.